model/activitylog.go

```diff
@@ -1,33 +1,78 @@
 package model
 
 import (
-	"time"
-
 	"golang.org/x/net/context"
 	"google.golang.org/appengine/datastore"
+	"time"
+	"types"
+	"util"
 )
 
 //TODO: need to add json tags for column names and to ignore blank fields
 type (
 	ActivityLog struct {
-		Name        string
-		Notes       string `json:"Notes,omitempty"`
-		StartTime   time.Time
-		EndTime     time.Time
-		CreatedDate time.Time
-		ModifiedOn  time.Time `json:"ModifiedOn,omitempty"`
```

Comment:
issue #2 with time.time is fixed. enable the StartTime and EndTime fields' functionality in struct and rest of the ActivityLog handlers